src/firmware_logger_device.cpp

```diff
@@ -44,91 +66,151 @@ namespace librealsense
 
     void firmware_logger_device::get_fw_logs_from_hw_monitor()
     {
-        auto res = _hw_monitor->send(_fw_logs_command);
-        if (res.empty())
+        command update_command = get_update_command();
+        if( update_command.cmd == 0 )
+            return;
+
+        auto res = _hw_monitor->send( update_command );
+        if( res.empty() )
         {
             return;
```

Comment:
would you want to add some log here?

Reply:
No, it is a valid case where the FW does not have any logs to report.